apps/condo/domains/division/access/Division.js

```diff
@@ -5,43 +5,41 @@
 const { throwAuthenticationError } = require('@condo/domains/common/utils/apolloErrorFormatter')
 const { checkOrganizationPermission, queryOrganizationEmployeeFor, queryOrganizationEmployeeFromRelatedOrganizationFor } = require('@condo/domains/organization/utils/accessSchema')
 const get = require('lodash/get')
-const { Division } = require('../utils/serverSchema')
+const { USER_SCHEMA_NAME } = require('@condo/domains/common/constants/utils')
+const { getById } = require('@core/keystone/schema')
 
-async function canReadDivisions ({ authentication: { item: user } }) {
-    if (!user) return throwAuthenticationError()
-    if (user.isAdmin || user.isSupport) return {}
-    const userId = user.id
-
-    return {
-        organization: {
-            OR: [
-                queryOrganizationEmployeeFor(userId),
-                queryOrganizationEmployeeFromRelatedOrganizationFor(userId),
-            ],
-        },
+async function canReadDivisions ({ authentication: { item, listKey } }) {
+    if (!listKey || !item) return throwAuthenticationError()
+    if (item.deletedAt) return false
+    if (listKey === USER_SCHEMA_NAME) {
+        if (item.isSupport || item.isAdmin) return {}
+        const userId = item.id
```

Comment:
redundant variable

Reply:
introduced variable for increasing meaning passing userId to organization querying is more obvious fo me 😄